### PR TITLE
TF 2827 Fix clicking reply before content is loaded

### PR DESCRIPTION
**Pull request by dab246**
## Issue

#2827 

## Solution

- Only display the `Reply`, `Reply all`, `Forward`, `Print` actions when the email content has been loaded successfully.

## Resolved

https://github.com/linagora/tmail-flutter/assets/80730648/3f37d187-0980-43d3-afc5-dec79e58485b

### CODE REVIEW

File: lib/features/email/domain/model/email_print.dart [hunk 1]
```diff
@@ -46,26 +46,28 @@ class EmailPrint with EquatableMixin {
   });
 
   factory EmailPrint.generate({
-    required PrintEmailAction printEmailAction,
+    required BuildContext context,
```
Comment (tddang-linagora) [inline lib/features/email/domain/model/email_print.dart:49]: I don't think we should pass framework's object to model layer. Instead, try to declare each String accordingly. E.g. If we need a String for appName, add:
```dart
required String appName
```

Comment (dab246) [reply lib/features/email/domain/model/email_print.dart]: good detect. done

**Review by tddang-linagora — APPROVED** [review]

**Comment by dab246** [comment]
@hoangdat please review it

**Review by tddang-linagora — APPROVED** [review]

**Comment by dab246** [comment]
Rebase to the `refactor` branch is complete

**Comment by hoangdat** [comment]
- [ ] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] Safari

File: lib/features/email/presentation/extensions/event_action_type_extension.dart [hunk 2]
```diff
@@ -0,0 +1,20 @@
+
+import 'package:flutter/material.dart';
+import 'package:tmail_ui_user/features/email/domain/model/event_action.dart';
+import 'package:tmail_ui_user/main/localizations/app_localizations.dart';
+
+extension EventActionTypeExtension on EventActionType {
+
+  String getLabelButton(BuildContext context) {
```
Comment (hoangdat) [inline lib/features/email/presentation/extensions/event_action_type_extension.dart:8]: did not see this function in use

**Comment by hoangdat** [comment]
- [ ] printed email not correct in text background color
Screenshot 2024-06-19 at 21 25 18

**Comment by tddang-linagora** [comment]
**Tested on**
- [ ] Chrome
- [x] Firefox
- [ ] Edge
- [ ] Safari

**Review by tddang-linagora — APPROVED** [review]

File: lib/features/email/presentation/extensions/event_action_type_extension.dart (hunk 2 above)
Comment (dab246) [reply lib/features/email/presentation/extensions/event_action_type_extension.dart:8]: Removed.

**Review by hoangdat — APPROVED** [review]

**Pull request merged**